WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -80,12 +108,12 @@ final public class InteractiveNotificationsManager: NSObject {
 
         switch action {
         case .CommentApprove:
-            approveCommentWithCommentID(commentID, siteID: siteID)
+            approveCommentWithCommentID(commentID, noteID: noteID, siteID: siteID)
         case .CommentLike:
-            likeCommentWithCommentID(commentID, siteID: siteID)
+            likeCommentWithCommentID(commentID, noteID: noteID, siteID: siteID)
         case .CommentReply:
             if let responseText = responseText {
-                replyToCommentWithCommentID(commentID, siteID: siteID, content: responseText)
+                replyToCommentWithCommentID(commentID, noteID: noteID, siteID: siteID, content: responseText)
```

Comment:
> I know this is not part of your PR! but since you're there, can we rename to a Swifty 3 style? 

There are a lot of methods that need renaming on `CommentService` and doing all of them will affect several files. I think we will be better doing that on a separate PR.